### PR TITLE
add spain

### PR DESCRIPTION
**Pull request by heavyk**

### CODE REVIEW

File: provinces.json [hunk 1]
```diff
@@ -941,6 +941,53 @@
 { "name": "Gran Chaco", "region": "Tarija", "country": "BO" },
 { "name": "José María Avilés", "region": "Tarija", "country": "BO" },
 
+{"short":"C","country":"ES","name":"La Coruña"},
```
Comment (azer) [inline provinces.json]: could you add space after & before brackets and after commas ?

Comment (heavyk) [inline provinces.json]: sure.. I only copy and pasted JSON.stringify :)

while I'm at it, do you need name to be first too?

Comment (azer) [inline provinces.json]: yeah would be nice to put the name on first, thanks

**Comment by heavyk** [comment]
done.

want me to format the rest with spaces too?

**Comment by azer** [comment]
thx

**Pull request merged**